packages/aurora-file-io/index.js

```diff
@@ -18,29 +18,22 @@ function save(note) {
 }
 
 /**
- * Asynchronously delete a note by it's ID
+ * Delete a note by it's ID
  */
 function deleteNote(id) {
   const context = getAuroraDirContext();
   deleteNoteFrom(id, context);
 }
 
 /**
- * Asynchronously load notes
+ * Loads notes one by one.
  */
 function loadNotes(callback) {
   const context = getAuroraDirContext();
   const noteFiles = context.find({ matching: "*" + noteFileExt });
-  const notes = [];
 
   noteFiles.forEach(file => {
-    context.readAsync(file).then(data => {
-      notes.push(NoteModel.fromFileData(data));
-      const allNotesAreLoaded = notes.length === noteFiles.length;
-      if (allNotesAreLoaded) {
-        callback(notes);
-      }
-    });
+    callback(NoteModel.fromFileData(context.read(file)));
```

Comment:
Is this actually faster? This is generally not a good idea because it blocks the thread.

Reply:
Welp, after pulling the branch and comparing it with a lot of notes, I guess it is much faster. I didn't realize the async overhead was so slow O.O